deploy/kubernetes/operator.nsx.vmware.com_ncpinstalls_crd.yaml

```diff
@@ -45,6 +45,14 @@ spec:
               addNodeTag:
                 description: Tag node logical switch port with node name and cluster when set to true, skip tagging when set to false
                 type: boolean
+              nsx-ncp:
+                description: NcpInstallnsx-ncp defines nodeSelector and tolerations for nsx-ncp pod Deployment
```

Comment:
"NcpInstallnsx-ncp" looks strange... And I think it's just an entry that allows the user to configure some NCP properties, nodeSelector and toleration are what it can contain, not equivalent to it. And if this patch didn't include the feature of toleration, I think we'd better not mention it for now. And "nsx-ncp deployemnt", not "nsx-ncp pod Deployment"